tests/lib/rules/require-prop-types.js

```diff
@@ -69,6 +69,24 @@ ruleTester.run('require-prop-types', rule, {
         }
       `,
       parserOptions: { ecmaVersion: 6, sourceType: 'module' }
+    },
+    {
+      filename: 'test.vue',
+      code: `
+        export default {
```

Comment:
It's needed to enclose by `<script>` element.

Reply:
Nope, it's not needed. It works good as is :) It's enough to put `export default {` and `.vue` filename :)

Reply:
Oh, I got it. I had misunderstood that the tests are using `vue-eslint-parser`.